pkg/virt-api/webhooks/validating-webhook/admitters/validators.go

```diff
@@ -0,0 +1,32 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2024 Red Hat, Inc.
+ *
+ */
+
+package admitters
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	k8sfield "k8s.io/apimachinery/pkg/util/validation/field"
+
+	v1 "kubevirt.io/api/core/v1"
+)
+
+type Validator interface {
+	// ValidateCreation validate the given spec on creation flow
+	ValidateCreation(field *k8sfield.Path, spec *v1.VirtualMachineInstanceSpec) []metav1.StatusCause
+}
```

Comment:
This interface defined in its own file because it used in multiple files as part for the same package.

> Considering the fact that in Go interfaces are implicit,

I am not sure what you mean by that.

> is there a reason to export it?

It should be exported because it used by other package:
Defined at `pkg/virt-api/webhooks/validating-webhook/admitters`
Used by `pkg/virt-api/webhooks/validating-webhook`